Dockerfile

```diff
@@ -4,11 +4,15 @@ WORKDIR /usr/src/databricks-cli
 
 COPY . .
 
-RUN pip install --upgrade pip && \
-    pip install -r dev-requirements.txt && \
```

Comment:
Is a minimal fix here just to also install tox-requirements?

Reply:
That is certainly the heart of the change -- and almost enough to fix the issue,

You also need to bump the versions on some of the packages in `tox-requirements.txt` or it will fail due to pytest-django and pyflakes dependencies. Also, you need to add the `tabulate` and `click` packages to `dev-requirements.txt`.

My other edits to Dockerfile were an attempt to minimize the resulting image size, but I didn't get a dramatic savings.

Reply:
Gotcha. I'm a bit concerned about changing the `dev-requirements.txt` and `tox-requirements.txt` to make this docker container build work.

I think a minimal fix is to intall `tox-requirements.txt` and to move up `pip install .` above the `./lint.sh`

Also it may be useful to add a `.dockerignore` file containing
```
**/__pycache__
**/*.pyc
```
so we don't run into https://stackoverflow.com/questions/44067609/getting-error-importmismatcherror-while-running-py-test

Reply:
with @andrewmchen proposals (add tox-requirements to pip install and moving ./lint.sh) works without problem. 

thanks